src/main/scala/bubblewrap/PageParser.scala

```diff
@@ -44,11 +44,8 @@ trait ContentType {
   )
 
   def isGzip(content: Content) = {
-    (content.contentEncoding, content.contentType) match {
-      case (Some(encoding), _) => gzipVariants.exists(gz => gz.equals(encoding))
-      case (_, Some(cType)) => gzipVariants.exists(gz => gz.equals(cType))
-      case (_, _) => false
-    }
+    val cType = Some(content.contentType.getOrElse("").split(";").head)
```

Comment:
`val cType: OPtion[String] = contentType.flatMap(_.split(";").headOption)`

Reply:
Fixed in 96c0f92.